### DISCUSSION TITLE
Missing openhab-js maintainers?

### DISCUSSION THREAD

**Discussion by florian-h05** [openhab/openhab-distro#1511] [openHAB Maintainers Discussions]
Dear @openhab/maintainers,

I have noticed that all openHAB repos have a maintainers group except the openhab-js repo.
openhab-js is quite new so may it is possible that there is no maintainers group because no one thought about it?
I would say @digitaldan and @jpg0 (maybe also me??) might be some candidates for it.

Many greetings,
Florian Hotze

**Comment by kaikreuzer** [comment]
Hi @florian-h05,

Good point, this was indeed missed when the repo was created.
I have now set up an according team on Github (https://github.com/orgs/openhab/teams/javascript-maintainers) and made @digitaldan the admin of it. As he was the one to have this repo created, I leave it to him to decide on the initial maintainers for the repo and have him add them to the team.

Best regards,
Kai
